src/__tests__/documenter.test.ts

```diff
@@ -6,5 +6,13 @@ import { getAllComponents, requireComponentDefinition } from "./utils";
 
 test.each<string>(getAllComponents())(`definition for %s matches the snapshot`, (componentName: string) => {
   const definition = requireComponentDefinition(componentName);
+
+  // overriding with a fake value so that when there are icon changes in components this test doesn't block it
+  if (componentName === "avatar") {
```

Comment:
I would skip this check as we can have the iconName prop added to more components